### DISCUSSION TITLE
What is "unmeshed node"?

### DISCUSSION THREAD

**Discussion by TalZiv** [linkerd/linkerd2#5153] [General]
Hi,
I am testing linkerd, I noticed that elastic client is not working properly and upon inspection i saw "unmeshed nodes"

by looking at the URL's i think that what I am looking at are the readiness probes of K8s failing...

Do i need to mesh the node? what does it mean?

![Screenshot from 2020-10-28 14 18 17](https://user-images.githubusercontent.com/22313897/97435211-cafe9400-1928-11eb-94ab-cbbb5d1015a9.jpg)

Thanks,
Tal

**Comment by grampelberg** [comment]
You can't mesh the node, it just means that traffic is coming in from a source that doesn't have a proxy.

**Reply by TalZiv** [reply]
Ok, what do i do with all the failed readiness and health probes? Does the mTLS causes this?  I installed linkerd with helm and provided a CA certificate. Thanks for the help

**Reply by grampelberg** [reply]
It shouldn't have anything to do with mTLS. What is in the logs?

**Reply by TalZiv** [reply]
I described the pods and in the events i see they are failing, can you point me to the log you are referring?

**Reply by grampelberg** [reply]
Your application logs and the proxy logs.
